calnex/api/api_test.go

```diff
@@ -760,3 +760,15 @@ func TestPushLicenseError(t *testing.T) {
 
 	require.Equal(t, originalLicense, uploadedLicense)
 }
+func TestReboot1(t *testing.T) {
+	// Set up test variables
+	target := "192.168.1.100"
```

Comment:
Please don't do that. Start https server like every other test is doing

Reply:
Kindly check the updated code

Reply:
Please squash commits (so you have only 1)